.eslintrc.json

```diff
@@ -11,27 +12,71 @@
 		"browser": true,
 		"node": true
 	},
+	"globals": { "Proxy": "readonly" },
+	"extends": [
+		"eslint:recommended",
```

Comment:
I’m a little uncomfortable with leaving coding style up to mystery bags of rules that we have little control over, and don't fully realize what we're enforcing. Could we list the rules explicitly?

Reply:
I'm torn on this. On one hand I understand wanting the rules to be explicit and opt-in; on the other hand ESLint is very stable, and maintains a long list of recommended rules that don't change often and would be duplicated here. And `typescript-eslint` also _disables_ some of the default ESLint rules that it replaces, so it would be a very long list. I'm open to copying them here if you prefer that (and we could try narrowing them down). The lists this is currently using are:

- https://github.com/eslint/eslint/blob/v8.54.0/packages/js/src/configs/eslint-recommended.js
- https://github.com/typescript-eslint/typescript-eslint/blob/v6.13.1/packages/eslint-plugin/src/configs/eslint-recommended.ts
- https://github.com/typescript-eslint/typescript-eslint/blob/v6.13.1/packages/eslint-plugin/src/configs/recommended.ts
- https://github.com/typescript-eslint/typescript-eslint/blob/v6.13.1/packages/eslint-plugin/src/configs/stylistic.ts

Reply:
Continued in #373